docs/cops_chefdeprecations.md

```diff
@@ -35,7 +35,7 @@ Exclude | `**/metadata.rb`, `**/Berksfile` | Array
 
 ### References
 
-* [https://rubystyle.guide#chefdkgenerators](https://rubystyle.guide#chefdkgenerators)
+* [https://rubystyle.guide#chefdeprecationschefdkgenerators](https://rubystyle.guide#chefdeprecationschefdkgenerators)
```

Comment:
I need to figure out why these aren't working. It should get the style guide from the department.